### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by zdy109** [deepnight/ldtk#670]
Stack:
```
LDtk version: 1.1.2-64bits
ENOENT: no such file or directory, scandir '/Applications/LDtk.app/Contents/MacOS/extraFiles/samples'
Error
Error: ENOENT: no such file or directory, scandir '/Applications/LDtk.app/Contents/MacOS/extraFiles/samples'
    at Object.readdirSync (node:fs:1390:3)
    at Object.e.readdirSync (node:electron/js2c/asar_bundle:5:10818)
    at new page_Home (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:187406:25)
    at file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:2873:12
    at App.loadPage (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:4015:25)
    at dn__$Delayer_Task.cb (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:2872:11)
    at dn_Delayer.update (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:64738:20)
    at Function.dn_Process.updateAll (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:499:14)
    at Boot.update (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:4572:14)
    at Boot.mainLoop (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:4534:8)
```
Processes:
```
#0 App
 |--- #1 page.Home
```

Log:
```


[BOOT]         App started
[BOOT]         Version: 1.1.2-64bits (build 458319)
[BOOT]         ExePath: /Applications/LDtk.app/Contents/MacOS
[BOOT]         Assets: /Applications/LDtk.app/Contents/Resources/app.asar/assets
[BOOT]         ExtraFiles: /Applications/LDtk.app/Contents/MacOS/extraFiles
[BOOT]         CWD: /
[BOOT]         Display: 1680x1050
[BOOT]         Args: soloValues=[], args=[]
[FILE]         Loading settings from /Users/zdy109/Library/Application Support/LDtk/settings...
[WARNING]        -> New settings
[BOOT]         AppZoomFactor: 1
[UPDATE]       Fetching latest version from GitHub...
[BOOT]         Calling appReady...
[BOOT]         Start args: path=null levelIndex=null
[BOOT]         Loading Home...
[GENERAL]      Page started: page.Home()
[FILE]         Loading page template: home from /Applications/LDtk.app/Contents/Resources/app.asar/assets/tpl/pages/home.html
[GENERAL]      Page started: page.CrashReport()
[FILE]         Loading page template: crashReport from /Applications/LDtk.app/Contents/Resources/app.asar/assets/tpl/pages/crashReport.html
[ERROR]        ENOENT: no such file or directory, scandir '/Applications/LDtk.app/Contents/MacOS/extraFiles/samples' (Error)
[ERROR]        Error: ENOENT: no such file or directory, scandir '/Applications/LDtk.app/Contents/MacOS/extraFiles/samples'
    at Object.readdirSync (node:fs:1390:3)
    at Object.e.readdirSync (node:electron/js2c/asar_bundle:5:10818)
    at new page_Home (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:187406:25)
    at file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:2873:12
    at App.loadPage (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:4015:25)
    at dn__$Delayer_Task.cb (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:2872:11)
    at dn_Delayer.update (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:64738:20)
    at Function.dn_Process.updateAll (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:499:14)
    at Boot.update (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:4572:14)
    at Boot.mainLoop (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:4534:8)

[GENERAL]      
#0 App
 |--- #1 page.Home
 |--- #2 page.CrashReport

[UPDATE]       Found 1.1.2
[UPDATE]       No new update.
```

**Comment by TheOnlyMrCat** [comment]
I got the same error. It looks like it's looking for a folder at `/Applications/LDtk.app/Contents/MacOS/extraFiles/samples`, but the real samples directory is in `/Applications/LDtk.app/Contents/extraFiles/samples` (skipping the `MacOS` folder). Manually moving the `extraFiles` folder into `MacOS` fixes this issue

**Comment by deepnight** [comment]
Sorry about this one, I'll make an update today.

**Comment by deepnight** [comment]
Fixed in 1.1.3
